packages/odyssey-storybook/src/guidelines/CustomTheming.stories.mdx

```diff
@@ -0,0 +1,107 @@
+import { Meta } from "@storybook/addon-docs/blocks";
+
+<Meta title="Guidelines/Custom Theming" />
+
+# Custom Theming Components with Odyssey
+
+Odyssey has its own theming system which other devs can hook into.
+
+While our internal system relies on Material-UI, you don't need to know any of that to interact with it.
+
+## Custom Component with Odyssey styles
+
+Customizing component styles with Odyssey theming is very simple. Just import `useTheme`, and everything is in there ready for you.
+
+```tsx
+import { css } from "@emotion/react";
+import { MouseEventHandler, ReactNode, useMemo } from "react";
+
+import { useTheme } from "./";
+
+export type WhatchamacallitProps = {
+  children: ReactNode;
+  onClick: MouseEventHandler;
+};
+
+export const Whatchamacallit = ({
+  children,
+  onClick,
+}: WhatchamacallitProps) => {
+  const theme = useTheme();
+
+  const whatchamacallitStyles = useMemo(
```

Comment:
How should one do this if it's going to be a component in `odyssey-react-mui` or `odyssey-labs`? 

For example, this will behave slightly differently from styles in `components.tsx`.

Reply:
It should perform the same for a consumer of Odyssey as Odyssey components themselves. The only difference is the import. `useTheme` comes from `./` instead of `@okta/odyssey-react-mui`.

Reply:
2e1dbf0746c6c583a531a18f8f798531864166d9

Fixed that import.